### PR TITLE
PR: Alert Component!

### PR DESCRIPTION
**Pull request by Jayesh3636**
- Added Alert Component
  - primary alert
  - secondary alert
  - success alert
  - warning alert
  - error alert

Please check out the alert component! let me know if there is any change 

https://user-images.githubusercontent.com/91378157/153618943-e0dc984c-62c8-483a-9cb5-0f2fdaf1a733.mp4

### CODE REVIEW

File: Components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    border-left: 10px solid darkorange;
+}
+
+.alert-error {
+    align-items: center;
+display: flex;
+    background-color: var(--alert-error-color);
+    color: darkred;
+    border-left: 10px solid darkred;
+}
```
Comment (ShoaibTamboli) [inline Components/alert/alert.css:108]: beautify/ Align the CSS file. Rest all looks good to me

**Review by ShoaibTamboli — COMMENTED** [review]
looks good to me.

File: Components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    border-left: 10px solid darkblue;
+}
+
+.alert-secondary {
+    align-items: center;
+display: flex;
+    background-color: var(--alert-secondary-color);
+    color: rgb(0, 0, 0);
+    
+    border-left: 10px solid;
```
Comment (Koushith) [inline Components/alert/alert.css]: you can use prettier extension to auto-format this indentation.

**Review by Koushith — APPROVED** [review]
LGTM. good to merge.

File: Components/alert/alert.css (hunk 1 above)
Comment (Jayesh3636) [reply Components/alert/alert.css:108]: done!

File: Components/alert/alert.css (hunk 2 above)
Comment (Jayesh3636) [reply Components/alert/alert.css]: done!

**Pull request merged**